korim/src/commonMain/kotlin/com/soywiz/korim/format/ImageFormat.kt

```diff
@@ -74,6 +74,8 @@ abstract class ImageFormat(vararg exts: String) : ImageFormatDecoder {
 
 	fun decode(s: SyncStream, props: ImageDecodingProps = ImageDecodingProps.DEFAULT): Bitmap = this.read(s, props)
 	//fun decode(file: File, props: ImageDecodingProps = ImageDecodingProps()) = this.read(file.openSync("r"), props.copy(filename = file.name))
+
+    // Decodes a given byte array to a bitmap based on the image format.
```

Comment:
To have documentation generation and shown in the IDE

```suggestion
    /** Decodes a given byte array to a bitmap based on the image format. */
```